webapp/src/components/ImageEditor/keyboardShortcuts.ts

```diff
@@ -1,6 +1,9 @@
-import { ImageEditorTool } from './store/imageReducer';
+
+import { Store } from 'react-redux';
+import { ImageEditorTool, ImageEditorStore } from './store/imageReducer';
 import { dispatchChangeZoom, dispatchUndoImageEdit, dispatchRedoImageEdit, dispatchChangeImageTool, dispatchSwapBackgroundForeground, dispatchChangeSelectedColor} from './actions/dispatch';
-import store from './store/imageStore';
+import mainStore from './store/imageStore';
```

Comment:
```suggestion
import { mainStore } from './store/imageStore';
```

I think should be that ^ re: lint ~